src/rocksdb/src/lib.rs

```diff
@@ -121,8 +120,11 @@ pub enum Error {
     TokioPanic(#[from] tokio::task::JoinError),
 }
 
-/// Options to configure a [`RocksDBInstance`].
-pub struct Options {
+/// Fixed options to configure a [`RocksDBInstance`].
+/// These are not tuning parameters, see the `tuning`
```

Comment:
Aren't you removing the `tuning` module in this PR?

Reply:
ah yes, its renamed to `config`! will update